packages/cozy-harvest-lib/src/components/KonnectorSuccess.jsx

```diff
@@ -0,0 +1,141 @@
+import has from 'lodash/has'
+import sortBy from 'lodash/sortBy'
+import get from 'lodash/get'
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
+
+import { ModalContent } from 'cozy-ui/transpiled/react/Modal'
+
+import Button from 'cozy-ui/react/Button'
+import withLocales from './hoc/withLocales'
+import DriveLink from '../components/KonnectorConfiguration/Success/DriveLink'
+import BanksLink from '../components/KonnectorConfiguration/Success/BanksLink'
+import connectingIllu from '../assets/connecting-data-in-progress.svg'
+import Markdown from './Markdown'
+
+const SuccessImage = () => <img src={connectingIllu} className="u-w-4 u-h-4" />
+
+const DescriptionContent = ({ title, message, children }) => {
+  return (
+    <>
+      <h4>{title}</h4>
+      <Markdown source={message} />
+      {children}
+    </>
+  )
+}
+
+export class KonnectorSuccess extends Component {
+  state = {
+    trigger: null
+  }
+  componentDidMount() {
+    const { accountId, accounts } = this.props
+    const matchingTrigger = get(
+      accounts.find(account => account.account._id === accountId),
+      'trigger'
+    )
+    this.setState({ trigger: matchingTrigger })
```

Comment:
Should it be derived state ? If the account rendered in this component changes the trigger will be stale.